Backend/utils/ResponseCreation.py

```diff
@@ -44,6 +48,8 @@ def prepForJsonDump(self, item):
       for i in iterator:
         #recursive call
         item[i] = self.prepForJsonDump(item[i])
+    elif isinstance(item, decimal.Decimal): #also numbers cant be dumped ??!
```

Comment:
Apparently json.dumps cannot handle the number for problem count (or anything else) unless it is first cast to a string